packages/cashier-v2/src/components/Clipboard/Clipboard.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import { useCopyToClipboard } from 'usehooks-ts';
+import { Tooltip, useDevice } from '@deriv-com/ui';
+import Copy from '../../assets/images/copy.svg';
+import CheckmarkCircle from '../../assets/images/ic-checkmark-circle.svg';
+import styles from './Clipboard.module.scss';
+
+type TProps = {
+    popoverAlignment: 'bottom' | 'left' | 'right' | 'top';
+    textCopy: string;
+};
+
+let timeoutClipboard: ReturnType<typeof setTimeout>;
```

Comment:
v.minor - this variable is gonna be shared among all instances of this component, which probably was not the intention. Though given the nature of component, I do not expect this to ever cause bugs.

Reply:
oh ,ok @wojciech-deriv got your point. will move this variable to component itself, in the next PR